### PR TITLE
Ground work for demo application

### PR DESCRIPTION
**Pull request by Lukasz825700516**
I prepared some ground work for the video handling. From this point it should be relatively easy to annotate graphically frames for show or something.

### CODE REVIEW

**Comment by Lukasz825700516** [comment]
[Wykrywanie w czasie rzeczywistym](https://trello.com/c/xtWuEQ5h/16-wykrywanie-w-czasie-rzeczywistym)

**Comment by Lukasz825700516** [comment]
[Dane z kamery](https://trello.com/c/t2feRjsY/18-dane-z-kamery)

File: scripts/yolop_zebra_crossing.py [hunk 1]
```diff
@@ -18,6 +30,9 @@ def main():
         frame = cv2.resize(frame, dsize=(640, 640), interpolation=cv2.INTER_CUBIC)
```
Comment (Iamhexi) [inline scripts/yolop_zebra_crossing.py]: Cubic interpolation is slow. You had better used linear interpolation.

Source: https://web.archive.org/web/20190424180810/http://tanbakuchi.com/posts/comparison-of-openv-interpolation-algorithms/

**Review by Iamhexi — CHANGES_REQUESTED** [review]

**Review by Iamhexi — APPROVED** [review]
Great job!

**Pull request merged**